src/styles/globals.ts

```diff
@@ -1,4 +1,6 @@
 import { css } from '@linaria/core'
+import '../../tokens/reapit/tokens.css'
```

Comment:
**note:** by using a bare module import like this, we enable our build tooling to be aware of this CSS module such that it will be processed and included in the final `index.css` artefact generated by a build.